content/en/tracing/trace_collection/trace_context_propagation/cpp.md

```diff
@@ -12,7 +12,7 @@ further_reading:
 
 ## Overview
 
-Datadog APM tracer supports [B3 headers extraction][11] and injection for distributed tracing.
+Datadog APM tracer supports [B3][11] headers extraction and injection for distributed tracing.
```

Comment:
This makes the mouse target very small (just `B3`) and harder to click. Consider making the link include `B3 headers`. I agree `extraction` should no be part of the link, though.

Reply:
Well, maybe not. I see now that there are many more instance of `B3` as a hyperlink below. I'd prefer to be consistent.